sys/Makefile.dep

```diff
@@ -718,4 +718,35 @@ ifneq (,$(filter dbgpin,$(USEMODULE)))
   FEATURES_REQUIRED += dbgpin
 endif
 
+ifneq (,$(filter fido2_ctap_%,$(USEMODULE)))
+  USEMODULE += fido2_ctap_transport
+  USEMODULE += fido2_ctap
+  ifneq (,$(filter fido2_ctap_hid,$(USEMODULE)))
+      ifneq (native, $(BOARD))
+        USEMODULE += usbus_hid
+        DISABLE_MODULE += auto_init_usbus
```

Comment:
Originally I had support for CTAPHID over UDP implemented to enable running the code with native target. Removed that functionality for now because it is not part of the official CTAP2 spec.

I forgot to remove this from the Makefile.dep. Will remove it.